object_detection:
  ros__parameters:
    input_img_topic: color_camera/image_raw
    output_bb_topic: object_detection/img_bb
    output_img_topic: object_detection/img
    model_params:
      detector_type: YOLOv8
      model_dir_path: /home/jayesh/ws/src/ros-perception-pipeline/object_detection/models
      weight_file_name: version5.pt
      confidence_threshold : 0.7
      show_fps : 1
